swagger.yml

```
@@ -6318,3 +6423,49 @@ components:
       flows:
         clientCredentials:
           tokenUrl: "https://sender.api.kivra.com/v1/auth"
+
+x-webhooks:
+  content.created:
+    post:
+      summary: Notification for new content
+      operationId: webhook-content-created
+      tags:
+        - "Partner API v3"
+      description: |
+        This webhook notifies you when a company inbox receives new content.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - id
+                - event_created_at
+                - event_type
+                - data_version
+                - data
+              properties:
+                id:
+                  $ref: "#/components/schemas/WebhookEventId"
+                event_created_at:
+                  $ref: "#/components/schemas/WebhookEventCreatedAt"
+                event_type:
+                  $ref: "#/components/schemas/WebhookEventType"
+                data_version:
+                  $ref: "#/components/schemas/WebhookDataVersion"
```

If the version changes it is because a breaking change was introduced, thus I think we have to document each supported version separately.

```suggestion
                event_type:
                  type: string
                  enum: ["content.created"]
                data_version:
                  type: integer
                  enum: [1]
```